charts/spire/charts/spire-server/values.yaml

```diff
@@ -91,6 +91,13 @@ ca_subject:
   organization: Example
   common_name: example.org
 
+nodeAttestor:
+  k8s:
+    psat:
+      enabled: true
+      rbac:
+        create: true
```

Comment:
I would not nest the psat part but do it like k8sPsat because that makes it easier to loop all keys once we add more nodeAttestors.

Having multiple levels of nesting complicates this.

Reply:
kk